src/preferences/dialog/dlgprefeffects.cpp

```diff
@@ -12,26 +13,32 @@ DlgPrefEffects::DlgPrefEffects(QWidget* pParent,
           m_pEffectsManager(pEffectsManager) {
     setupUi(this);
 
-    connect(availableEffectsList,
-            SIGNAL(currentItemChanged(QListWidgetItem*, QListWidgetItem*)),
+    m_pAvailableEffectsModel = new EffectSettingsModel();
+
+    m_pAvailableEffectsModel->resetFromEffectManager(pEffectsManager);
+    availableEffectsList->setModel(m_pAvailableEffectsModel);
+
+    connect(availableEffectsList->selectionModel(),
+            SIGNAL(currentRowChanged(const QModelIndex&, const QModelIndex&)),
             this,
-            SLOT(slotEffectSelected(QListWidgetItem*, QListWidgetItem*)));
+            SLOT(availableEffectsListItemSelected(const QModelIndex&)));
+
+    // Highlight first row
+    availableEffectsList->selectRow(0);
+
+    availableEffectsList->setColumnWidth(0, 50);
```

Comment:
The width of the first column cuts off the "Visible" column header text for me (Qt5 with 2.5 scale factor)
![screenshot from 2018-05-20 08-17-33](https://user-images.githubusercontent.com/9455094/40279330-44c5638c-5c06-11e8-8516-cff5042ea737.png)